includes/classes/Blocks.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Jobber Blocks.
+ *
+ * @package Jobber
+ */
+
+namespace Jobber;
+
+use TenupFramework\Module;
+
+/**
+ * Base class for Jobber Blocks
+ */
+class Blocks {
+	use Module;
+
+	/**
+	 * Can we register this module?
+	 *
+	 * @return bool
+	 */
+	public function can_register(): bool {
+		return true;
+	}
+
+	/**
+	 * Hook the module into WP.
+	 */
+	public function register() {
+		add_action( 'init', [ $this, 'register_block_types' ] );
+	}
+
+	/**
+	 * Register the block types.
+	 */
+	public function register_block_types() {
+		register_block_type(
```

Comment:
Would it be better to check if we have a valid access token before we register the block? Right now I can add the block immediately after activating the plugin, though it will show an error message. I guess I can see two approaches:

1. Don't register the block at all until an account has been connected
2. Register the block but update our error message in the admin to point people to the settings page so they can authenticate